### ISSUE TITLE
strange case of marks

### ISSUE THREAD

**Issue by arademaker** [own-pt/sensetion.el#122]
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/86403/60980328-a719d800-a30a-11e9-9ad3-4103a24d7ff9.png">

why dark-brown has the superscript JJ1?

**Comment by arademaker** [comment]
It should be `a1` as other single word tokens?

**Comment by arademaker** [comment]
why MWE does not follow the presentation of single tokens that, once annotated, have the superscript on the right with `pos+buffer-sense` where `buffer-sense` is a single-digited sense in the buffer.

**Comment by odanoburu** [comment]
does https://github.com/own-pt/sensetion.el/commit/3144dc1d8483e6ac4d45b20693a26b63585eb641 fix the problem?

although I made it show `s1` as it's an adjective satellite; or do you think we should collapse it to adjective? (I guess annotation-wise it makes more sense, as satellites are not a real PoS..)

> why MWE does not follow the presentation of single tokens that, once annotated, have the superscript on the right with pos+buffer-sense where buffer-sense is a single-digited sense in the buffer.

not sure I understand what you mean, but could it be that the problem is that `dark brown` is a glob, but `dark-brown` is not?

**Comment by odanoburu** [comment]
can I close this, @arademaker ?

**Comment by arademaker** [comment]
No. In the buffer `any@dark-brown` the superscripts are only numbers. In 

> (n) anthophyllite | a dark brown mineral of the amphibole group ; magnesium iron silicate ; 

the glob `drak-brown` should have the superscript `a1`.

**Comment by odanoburu** [comment]
this should be fixed now in dev by https://github.com/own-pt/sensetion.el/commit/4b168e24b757e3b61db3bd3bcd6980402daf5d55

**Comment by arademaker** [comment]
this fixes the code in dev branch, right?

**Comment by odanoburu** [comment]
fd2f53877e20c4c4317a57f29cc8d8b16273fa61 fixes it in master too.

**Comment by odanoburu** [comment]
must show PoS of globs too.
